app/util/notifications/notification-states/lido-withdrawal-requested/lido-withdrawal-requested.tsx

```diff
@@ -0,0 +1,88 @@
+import { strings } from '../../../../../locales/i18n';
+import { TRIGGER_TYPES } from '../../constants';
+import { ExtractedNotification, isOfTypeNodeGuard } from '../node-guard';
+import { NotificationState } from '../types/NotificationState';
+import {
+  getAmount,
+  getNativeTokenDetailsByChainId,
+  getNotificationBadge,
+} from '../../notification.util';
+import { getTokenAmount } from '../token-amounts';
+
+type LidoWithdrawalRequestedNotification =
+  ExtractedNotification<TRIGGER_TYPES.LIDO_WITHDRAWAL_REQUESTED>;
+
+const isLidoWithdrawalRequestedNotification = isOfTypeNodeGuard([
+  TRIGGER_TYPES.LIDO_WITHDRAWAL_REQUESTED,
+]);
+
+const state: NotificationState<LidoWithdrawalRequestedNotification> = {
+  guardFn: isLidoWithdrawalRequestedNotification,
+  createMenuItem: (notification) => {
+    const amount = `${getAmount(
+      notification.data.stake_in.amount,
+      notification.data.stake_in.decimals,
+      { shouldEllipse: true },
+    )}`;
```

Comment:
Q: why the string backticks? it looks like it could be directly returned as a value:
```suggestion
    const amount = getAmount(
      notification.data.stake_in.amount,
      notification.data.stake_in.decimals,
      { shouldEllipse: true },
    );
```

Reply:
Addressed.